packages/edit-site/src/components/header/style.scss

```diff
@@ -20,6 +20,13 @@
 	}
 }
 
+// Keeps the document title centered when the sidebar is open
+.navigation-open {
+	.edit-site-header_center {
+		margin-left: $nav-sidebar-width;
```

Comment:
**Note:**

Still seeing a little bit of wonkiness when we shrink the browser viewport with the sidebar open

Reply:
It seems like the current solution doesn't play well when shrinking the viewport 🤔

Reply:
ah yeah. it looks fine when the sidebar is closed, but is a lot more sticky when it's open

Reply:
Yeah currently mulling over the best way to approach this. I think we might have to make tradeoffs between absolute positioning, which causes [this](https://github.com/WordPress/gutenberg/pull/25630#pullrequestreview-496529493), and adding padding to the document title, which causes the "center div to be pushed over further than it should be when the distance between "left" and "right" is small enough".